paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -65,6 +65,18 @@ internal abstract class BaseSheetViewModel(
     private val _paymentMethodMetadata = MutableStateFlow<PaymentMethodMetadata?>(null)
     internal val paymentMethodMetadata: StateFlow<PaymentMethodMetadata?> = _paymentMethodMetadata
 
+    private val _hasSeenDirectToCardScan = MutableStateFlow(false)
+    internal val hasSeenDirectToCardScan: StateFlow<Boolean> = _hasSeenDirectToCardScan
+
+    internal val shouldOpenCardScanAutomatically: StateFlow<Boolean> = combineAsStateFlow(
```

Comment:
I don't think we'll want to do this.

We probably want to use a SavedStateHandle, and just initialize the value with paymentMethodMetadata.